src/components/DistanceRequest/index.js

```diff
@@ -238,7 +271,7 @@ function DistanceRequest({transactionID, report, transaction, route, isEditingRe
         </>
     );
 
-    if (!isEditing) {
+    if (!isEditingNewRequest) {
```

Comment:
I don't understand this change, would you please change it and include a comment in the code?

Reply:
Ok yes I understand isEditingNewRequest is true when we click the distance field from the confirmation screen while creating a new request.  This condition is deciding whether to return only content, or to wrap it with a screen wrapper and header. We don't need the header for the new distance request page, nor for the editing request page, but we are missing a header when editing a new distance request with the current change 😕 
![image](https://github.com/Expensify/App/assets/26260477/77e238eb-0a57-481f-bd66-50693adf8a58)

Reply:
@dukenv0307 please fix this bug